app/models/item.js

```diff
@@ -124,12 +128,8 @@ export default cloudinaryUrl.extend({
     }
   ),
 
-  availableQty: Ember.computed("lockedQty", "receivedQuantity", function() {
-    return this.get("receivedQuantity") - this.get("lockedQty");
-  }),
-
-  isAvailable: Ember.computed("availableQty", function() {
-    return this.get("availableQty") > 0;
+  isAvailable: Ember.computed("availableQuantity", function() {
+    return this.get("availableQuantity") > 0;
   }),
```

Comment:
https://api.emberjs.com/ember/2.15/namespaces/Ember.computed/methods?anchor=bool

```suggestion
  isAvailable: Ember.computed.bool("availableQuantity");
```